styles/components/LatestNews.module.scss

```diff
@@ -0,0 +1,38 @@
+.latestNews {
+  @apply bg-color-4 grid;
+
+  .newsItem {
+    @apply grid grid-cols-5 items-center;
+    @apply m-4 p-4;
+    @apply rounded-xl;
+    &:hover {
+      @apply bg-hover-1;
+      @apply shadow-lg;
+      @apply transition-all;
+      @apply duration-300 ease-in-out;
```

Comment:
Move this to the parent, you change the item on hover, the transition stuff always stays.

Reply:
fixed